libs/ledgerjs/packages/hw-transport/src/Transport.ts

```diff
@@ -120,6 +120,40 @@ export default class Transport {
     throw new Error("exchange not implemented");
   }
 
+  /**
+   * Send apdus in batch to the device using a low level API.
+   * The default implementation is to call exchange for each apdu.
+   * @param {Array<Buffer>} apdus - array of apdus to send.
+   * @param Observer<Buffer> observer - an observer that will receive the response of each apdu.
```

Comment:
```suggestion
   * @param {Observer<Buffer>} observer - an observer that will receive the response of each apdu.
```

If we want it to be correctly formatted ;)